pkg/controller/nodelifecycle/node_lifecycle_controller_test.go

```diff
@@ -91,7 +91,11 @@ func (nc *nodeLifecycleController) doEviction(fakeNodeHandler *testutil.FakeNode
 	for _, zone := range zones {
 		nc.zonePodEvictor[zone].Try(func(value scheduler.TimedValue) (bool, time.Duration) {
 			uid, _ := value.UID.(string)
-			nodeutil.DeletePods(fakeNodeHandler, nc.recorder, value.Value, uid, nc.daemonSetStore)
+			pods, err := listPodsFromNode(nc.kubeClient, value.Value)
```

Comment:
fakeNodeHandler implements clientset.Interface

Either change here to use fakeNodeHandler here or use nc.kubeClient both here and below. I guess the former seem better.

Reply:
Same below

Reply:
Done.